codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2StreamChannel.java

```diff
@@ -105,6 +105,28 @@ public Handle newHandle() {
     private static final AtomicIntegerFieldUpdater<AbstractHttp2StreamChannel> UNWRITABLE_UPDATER =
             AtomicIntegerFieldUpdater.newUpdater(AbstractHttp2StreamChannel.class, "unwritable");
 
+    private final ChannelFutureListener windowUpdateFrameListener = new ChannelFutureListener() {
+        @Override
+        public void operationComplete(ChannelFuture future) {
+            handleWindowUpdateFrameFuture(future);
+        }
+    };
+
+    private void handleWindowUpdateFrameFuture(ChannelFuture future) {
+        Throwable cause = future.cause();
+        if (cause != null) {
+            cause.printStackTrace();
```

Comment:
Maybe log this instead of dumping it to stderr

Reply:
actually this is a left-over from debugging. This should are just removed.